docs/main.md

```diff
@@ -55,11 +55,11 @@ To trigger the OIDC authentication flow, a client should issue a GET request to
 Local authentication uses username/password 
 Mosip uses Foundational ID (VID)
 
-| VID / username                       | Role               | subject                              | Description                                         |
-|--------------------------------------|--------------------|--------------------------------------|-----------------------------------------------------|
-| 5649650687 / registry-administration | ENROLLMENT_OFFICER | 299950323465436931629862208523254959 | Officer responsible for enrollment                  |
```

Comment:
564965068 is for enrollment officer
4893724702 is for payment officer
2371487382 is for registry officer

Reply:
Implemented it 
https://github.com/GovStackWorkingGroup/sandbox-usecase-usct-backend/pull/44